modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java

```diff
@@ -812,8 +812,8 @@ private DDMFormValues _getDDMFormValues(
 			rootDDMField.getFieldId());
 
 		for (DDMFieldAttributeInfo ddmFieldAttributeInfo :
-				rootDDMFieldInfo._ddmFieldAttributeInfos.get(
-					StringPool.BLANK)) {
+				rootDDMFieldInfo._ddmFieldAttributeInfos.getOrDefault(
+					StringPool.BLANK, Collections.emptyList())) {
```

Comment:
@algobob Though I don't fully understand why this resolves the issue, the code looks good. 

If it resolves the customer's issue, please proceed with the hotfix. I'll close this PR now so you can concentrate on the tests.